examples/create-react-app/config-overrides.js

```diff
@@ -1,5 +1,5 @@
 const {getBabelLoader} = require('react-app-rewired')
-module.exports = (config, env) => {
+module.exports = (config, _env) => {
```

Comment:
Unused arguments will be underscored to avoid eslint errors.
https://eslint.org/docs/rules/no-unused-vars#argsignorepattern